google/resource_bigquery_dataset.go

```diff
@@ -228,6 +308,7 @@ func resourceBigQueryDatasetRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("project", id.Project)
 	d.Set("etag", res.Etag)
 	d.Set("labels", res.Labels)
+	d.Set("access", flattenAccess(res.Access))
```

Comment:
for setting values that aren't primitives, we're trying to get in the habit of checking the error returned from d.Set (we've had some confusing bugs in the past that would have shown up much sooner if we had error-checked). something like:
```
if err := d.Set("access", flattenAccess(res.Access)); err != nil {
  return err
}
```